tools/cliupdate.php

```diff
@@ -214,16 +214,29 @@ function displayWarning($msg, $red=false) {
    $glpilanguage        = $configurationValues['language'];
 }
 
-$migration = new CliMigration(GLPI_VERSION);
+$migration = new CliMigration(GLPI_SCHEMA_VERSION);
 
 $migration->displayWarning("Current GLPI Data version: $current_version");
+$migration->displayWarning("New     GLPI Data version: ".GLPI_SCHEMA_VERSION);
 $migration->displayWarning("Current GLPI Code version: ".GLPI_VERSION);
 $migration->displayWarning("Default GLPI Language: $glpilanguage");
 
 
 // To prevent problem of execution time
 ini_set("max_execution_time", "0");
 
+if (defined('GLPI_PREVER')) {
+   if (strlen($current_version) > 40 && $current_version != GLPI_SCHEMA_VERSION) {
+      if (!isset($args['force'])) {
+         die("Upgrading from a dev version to a dev version is not supported. Please upgrade manually - or add --force.");
```

Comment:
I think this should be supported (at least when code is same version, ie "9.2-dev"

IMHO this is the goal of this feature, "detect schema change and manage update"

By specifications, "update" have be indempotent, can be run various time without change.